admin_manual/configuration/files/external_storage/ftp.rst

```diff
@@ -2,13 +2,19 @@
 FTP/FTPS
 ========
 
+If you want to mount a FTP Storage, please install the app ``FTP Storage Support`` from ownCloud market.
```

Comment:
Can you change this sentence to: "If you want to mount an FTP Storage, please install `the FTP Storage Support app`_  from the ownCloud market." and link to the FTP Storage Support app in the marketplace?